codec_specs.md

```diff
@@ -71,7 +71,9 @@ which **MUST** be stored within the `CodecPrivate Element`. When the Initializat
 within a track, then that updated Initialization data **MUST** be written into the `CodecState Element`
 of the first `Cluster` to require it. If the encoding does not require any form of Initialization,
 then `none` **MUST** be used to define the Initialization and the `CodecPrivate Element`
-**SHOULD NOT** be written and **MUST** be ignored. Data that is defined Initialization to be
+**SHOULD NOT** be written and **MUST** be ignored. If the encoding does require any form of Initialization,
+then `CodecPrivate Element` **MUST** be written and **MUST** be provided to the decoder.
+Data that is defined Initialization to be
```

Comment:
I know that this sentence wasn't changed, but to be honest, I didn't like the original anyway. Maybe we can change it right now, too. On the other hand changing verbiage might be better handled as a separate MR.

If we wanted to change it now, the following might be better:

> The Initialization data to be stored in the `CodecPrivate Element` is referred to as `Private Data`.

Again, I'd be fine accepting this MR without this particular change.

Reply:
> If we wanted to change it now, the following might be better

Obvious bad typos, I fixed them in the same commit. Please review again.